documentation/CHANGES.rst

```diff
@@ -369,8 +369,8 @@ Changes to Existing Features
 
 - The following API elements of oneDPL were changed or removed:
 
-  - the ``default_policy`` object was renamed to ``dpcpp_default``;
-  - the ``fpga_policy`` object was renamed to ``dpcpp_fpga``;
+  - the ``default_policy`` object was renamed to ``dpcpp_default`` and declared as ``const``;
+  - the ``fpga_policy`` object was renamed to ``dpcpp_fpga`` and declared as ``const``;
```

Comment:
No, old release notes are a wrong place for this new information.

Reply:
Fixed (rolled back).